### DISCUSSION TITLE
Prime One Aura theme can change to Lara?

### DISCUSSION THREAD

**Discussion by evonleee** [primefaces/community#3026] [Figma UI Kit] [answered]
Hi, im planning to buy this UI Kit https://primevue.org/uikit/. I want to ask if I can change the theme to Lara as my team planned to use the Lara theme.

**Comment by onursenture** [comment]
Hello, Currently, only Aura is included in the new version of the UI Kit. Lara will be added in future versions.

**Reply by lauramenmen** [reply]
do you might happen to have a rough estimate when Lara will be added?

**Reply by onursenture** [reply, marked as answer]
> do you might happen to have a rough estimate when Lara will be added?

We have halted our plans to add Lara for now. Whether Lara will appear in future versions is uncertain; it’s not clearly defined on our roadmap.
